app.py

```diff
@@ -452,6 +454,25 @@ def get_first_name(schedule):
     return schedule.get("preferred_name") or schedule["firstname"]
 
 
+def get_github_info():
```

Comment:
call this `get_latest_github_commits` or something similar to make it clear what this function does